aten/src/ATen/native/cpu/UnaryOpsKernel.cpp

```diff
@@ -52,94 +54,59 @@ static void parallel_apply(Tensor& result, const Tensor& self, F f) {
 
 static void abs_kernel(Tensor& result, const Tensor& self) {
   AT_DISPATCH_ALL_TYPES(self.type(), "abs", [&] {
-    parallel_apply<scalar_t>(result, self, [](const Vec256<scalar_t>& x) {
-      return x.abs();
-    });
-  });
+    parallel_apply<scalar_t>(
+        result,
+        self,
+        [](const Vec256<scalar_t>& x) { return x.abs(); });  });
```

Comment:
Can you please try to limit formatting-only changes in the future? It makes reviewing much easier.

Reply:
That's difficult because I rely on clang format a lot for efficiency. Otherwise I'll spend a lot of time adjusting brackets and such after code mods.

Reply:
That’s difficult because I rely on diffs a lot for efficiency. Otherwise I spend a lot of time analyzing all the code, and I need to do it carefully to see if just unnecessary formatting, or if there was a real change ;)

As you see it’s a double edged sword, and we generally don’t have strict style rules. I’m ok with clang format but please run it only on the lines you actually edit. Thanks!

Reply:
Will do!